docs/pages/config-plugins/plugins-and-mods.mdx

```diff
@@ -65,7 +65,7 @@ You may want to create a plugin in a different file, here's how:
 Consider the following example that changes the config name:
 
 ```
-╭── app.config.js ➡️ Expo Config
+╭── app.config.js ➡️ app Config
```

Comment:
```suggestion
╭── app.config.js ➡️ App config
```

Casing nit. 

Btw. I would look at adding an ability to `FileTree` component to include comments for files, so we can use it in places like that.